jfix-stdlib-batching/src/main/java/ru/fix/stdlib/batching/BatchingManagerThread.java

```diff
@@ -114,6 +112,7 @@ public void run() {
 
             log.trace("Wait available thread in BatchProcessor thread pool");
             boolean isBatchProcessorThreadAvailable = false;
+            ProfiledCall threadAwaitCall = profiler.start("BatchingManagerThread_batch_processor_thread_await");
```

Comment:
Should we add batchManagerId in this metric name?

Reply:
Yes, we should. Will fix it shortly